e2e/windows/README.md

```diff
@@ -0,0 +1,73 @@
+- **e2e-create-windows-nodepool.sh**. It is the bash file used to add a test windows nodepool to the test cluster.
+- **e2e-helper.sh**. It is the help file that provides functions for other bash files.
+- **e2e-scenario.sh**. It is the bash file used to deploy a test vmss with the lastest windows cse package. If the deployment succeeds, we will continue to deploy a test pod to validate whether the node is successfully joined to the test cluster.
+- **e2e-starter.sh**. It is the bash file used to create the test cluster. After the cluster is first created, we will upload some files(i.e., certificate files) in the linux nodepool to the storage account. In this way, all the pipelines that use the same cluster to do the test could download the linux files and obtain needed parameters from them.
+- **e2e_test.go**. This file is used to generate CustomData file and CSEcmd file for further deployment of test vmss.
+- **matrix.json**. This json file is used to set parameters of SCENARIO_NAME and VM_SKU. Since VM_SKU is set in other code now, we can delete this file after setting SCENARIO_NAME in the code in the future.
+- **nodebootstrapping_template.json**. It is the first part of template used to generate windows CustomData file and CSECmd file and it contains the basic properties for the cluster and linux nodepool.
+- **percluster_template.json**. It is the second part of template used to generate windows CustomData file and CSECmd file and it contains properties that should be obtained from pipeline variables and linux files uploaded to the storage account.
+- **pod-windows-template.yaml**. This pod yaml file is used to deploy a test pod to validate whether the node is already joined to the test cluster.
+- **scenarios/windows/property-windows-template.json**. It is the third and last part of template used to generate windows CustomData file and CSECmd file and it contains properties that are related to the test windows nodepool.
+- **upload-cse-logs.ps1**. This script is used to upload generated cse logs to the stroage account in the deployed windows vmss for further download.
+- **windows_vmss_template.json**. This json file is the basic template used to deploy a windows vmss. We add necessary properties to this file in `e2e-scenario.sh`.
+
+## Generate and Update current windows test images
+Currently, we use test images "/subscriptions/$SUBSCRIPTION_ID/resourceGroups/akswinvhdbuilderrg/providers/Microsoft.Compute/galleries/AKSWindows/images/windows-e2e-test-$WINDOWS_E2E_IMAGE/versions/2023.02.07/" ($WINDOWS_E2E_IMAGE could be 2019-containerd, 2022-containerd, 2022-containerd-gen2) as is referred by `IMAGE_REFERENCE` in `e2e-scenario.sh`.
+
+To generate new windows test images, we can use [`AKS Windows VHD Build - PR check-in gate`](https://msazure.visualstudio.com/CloudNativeCompute/_build?definitionId=182855&_a=summary). Before running the pipeline, we need to set the values of pipeline variables as follows:
```

Comment:
should we include the pipeline link here as I think it's also sensitive info?

Reply:
This pipeline could also be clicked in the following checks, so I think it's ok here.